problems/totalNumberOfStudents.js

```diff
@@ -11,8 +11,13 @@
 * returns: 167
 */
 
-function totalNumberOfStudents() {
+const totalNumberOfStudents = (arr) => {
+    let sum = 0;
+    for(const studentCount in arr){
+     sum += arr[studentCount];
 
+    }
```

Comment:
Very close, when we use the `for in` with arrays your `studentCount` is the index of the array and you used it correctly in line 17 but you forgot to access the object's key. Currently the way you have it here you are just adding up objects.